packages/storybook/stories/ChatComposite/CustomDateTimeFormat.stories.tsx

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { ChatComposite, COMPONENT_LOCALE_EN_US, COMPOSITE_LOCALE_EN_US } from '@azure/communication-react';
+import { Stack } from '@fluentui/react';
+import { Meta } from '@storybook/react/types-6-0';
+import React, { useState, useEffect } from 'react';
+import { SingleLineBetaBanner } from '../BetaBanners/SingleLineBetaBanner';
+import { COMPOSITE_FOLDER_PREFIX, compositeExperienceContainerStyle } from '../constants';
+import { defaultChatCompositeHiddenControls, controlsToAdd } from '../controlsUtils';
+import { compositeLocale } from '../localizationUtils';
+import { getDocs } from './ChatCompositeDocs';
+import { ContosoChatContainer } from './snippets/Container.snippet';
+import {
+  ChatCompositeSetupProps,
+  ConfigHintBanner,
+  addParrotBotToThread,
+  createThreadAndAddUser
+} from './snippets/Utils';
+
+const messageArray = [
+  'Welcome to an example on how to add powerful customizations to the ChatComposite',
+  'By following this example, Contoso can customize the date time format to whatever they want!',
+  'Have fun!'
+];
+
+const CustomDateTimeFormatStory = (args, context): JSX.Element => {
+  const {
+    globals: { locale }
+  } = context;
+  const [containerProps, setContainerProps] = useState<ChatCompositeSetupProps>();
+
+  const onDisplayDateTimeString = (messageDate: Date): string => {
```

Comment:
Please move this function to file-level private function. We should avoid creating functions nested inside other functions because they reduce readability. You need to do this if you are capturing some local variable, even there it's better to create a helper function with most of the logic and call that from your closure defined here.